src/analysis/bounded_variation.lean

```diff
@@ -207,6 +207,56 @@ begin
   simp [u, edist_comm],
 end
 
+lemma lower_continuous_aux {ι : Type*} {F : ι → α → E} {p : filter ι}
+  {f : α → E} {s : set α} (hF : tendsto_uniformly_on F f p s) :
+  ∀ (v : ℝ≥0∞), v < evariation_on f s → (∀ᶠ (n : ι) in p, v < evariation_on (F n) s) :=
+begin
+  rintro v hv,
+  rw emetric.tendsto_uniformly_on_iff at hF,
+  obtain ⟨v',vv',v'var⟩ := exists_between hv,
+  obtain ⟨⟨n,⟨u,um,us⟩⟩,hlt⟩ := lt_supr_iff.mp v'var,
```

Comment:
```suggestion
  obtain ⟨v', vv', v'var⟩ := exists_between hv,
  obtain ⟨⟨n, ⟨u, um, us⟩⟩, hlt⟩ := lt_supr_iff.mp v'var,
```
Spaces after commas. Also, `obtain` becomes much more readable if you specify the type, as in 
```
  obtain ⟨v', vv', v'var⟩ : ∃ v', v < v' ∧ v' < evariation_on f s := exists_between hv,
```